cookbook/bundles/tests.rst

```diff
@@ -0,0 +1,311 @@
+.. index::
+   single: Bundle; Testing third party bundles
+
+Writing Tests for Third Party Bundles
+=====================================
+
+The Standard Edition's front controller looks like this::
```

Comment:
We usually have a little introduction paragraph at the start of the article, instead of starting with the text directly. What about adding such a paragraph here as well? For instance:

 > When creating a shared bundle, you need to verify if the bundle works as it should. The best practice is to write automatic functional tests. This article will guide you through setting up and running such tests.